### ISSUE TITLE
Support libssl.so.47

### ISSUE THREAD

**Issue by NanXiao** [dom96/choosenim#134]
On my `Void Linux`, `libssl` version is `libssl.so.47`:

```
$ ls -lt /usr/lib/libssl*
lrwxrwxrwx 1 root root     16 Jun 12 20:06 /usr/lib/libssl.so.47 -> libssl.so.47.0.5
-rw-r--r-- 1 root root 365472 Jun 12 20:06 /usr/lib/libssl.so.47.0.5
```

Running `choosenim` will report following errors:

```
$ curl https://nim-lang.org/choosenim/init.sh -sSf | sh
choosenim-init: Downloading choosenim-0.4.0_linux_amd64
could not load: libssl.so(.1.1|.1.0.2|.1.0.1|.1.0.0|.0.9.9|.0.9.8|.46|.45|.44|.43|.41|.39|.38|.10|)
compile with -d:nimDebugDlOpen for more information
```

Is it possible to support `libssl.so.47`? Thanks!

**Comment by SolitudeSF** [comment]
it is already supported, choosenim would just have to be rebuilt with newer compiler.

**Comment by genotrance** [comment]
Can you please check if this is fixed in v0.5.0?

**Comment by dom96** [comment]
I'm going to close pre-emptively assuming that it is. Please reopen if it's still broken.
